Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 import MediaPlayer
 
-class AttachmentApprovalViewController: UIViewController, OWSAudioAttachmentPlayerDelegate {
+class AttachmentApprovalViewController: OWSViewController, OWSAudioAttachmentPlayerDelegate {
```

Comment:
I added a `OWSViewController` base class that logs view deallocation.  It helps expose these leaks, which are easy to do.